src/components/ImagePreviewCard/ImagePreviewCard.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Box,
+  Center,
+  chakra,
+  Grid,
+  GridItem,
+  Image,
+  Text,
+  useMultiStyleConfig,
+  VStack,
+} from "@chakra-ui/react"
+import { Checkbox } from "@opengovsg/design-system-react"
+import { ChangeEvent, useState } from "react"
+import { BiEditAlt, BiTrash } from "react-icons/bi"
+
+import { ContextMenu } from "components/ContextMenu"
+
+import { convertUtcToTimeDiff } from "utils/dateUtils"
+
+import { CARD_THEME_KEY } from "theme/components/Card"
+
+export interface ImagePreviewCardProps {
+  name: string
+  addedTime: number
+  mediaUrl: string
+  isMenuNeeded?: boolean
+  onCheck?: (event: ChangeEvent<HTMLInputElement>) => void
+}
+
+// Note: This is written as a separate component as the current Card API is not
+// flexible enough to support this use case.
+export const ImagePreviewCard = ({
+  name,
+  addedTime,
+  mediaUrl,
+  isMenuNeeded = true,
+  onCheck,
+}: ImagePreviewCardProps): JSX.Element => {
+  const [isSelected, setIsSelected] = useState(false)
+  const styles = useMultiStyleConfig(CARD_THEME_KEY, {})
+  const relativeTime = convertUtcToTimeDiff(addedTime)
+
+  return (
+    <Box position="relative" h="100%" data-group>
```

Comment:
have we tried using the `Card` component given by chakra? if you haven't then it's fine - just wanted to find out if it's easy to migrate to the `Card` component so we have less of these 1 off stuff

Reply:
Ah RIP I didn't notice that Chakra had their own Card component. I think yes it should be possible to move over but going to take this L for now.